### PR TITLE
Update SpringBoot@2.6.14

### PR DESCRIPTION
**Pull request by mcornaton**

### CODE REVIEW

**Review by jbern0rd — APPROVED** [review]
Thanks !

File: build.gradle [hunk 1]
```diff
@@ -43,7 +43,7 @@ repositories {
 // 'api' configuration allows to expose dependencies with 'compile' scope in pom
 // 'implementation' configuration allows to expose dependencies with 'runtime' scope in pom
 dependencies {
-    implementation platform("org.springframework.boot:spring-boot-dependencies:2.6.2")
+    implementation platform("org.springframework.boot:spring-boot-dependencies:2.6.14")
```
Comment (james-toussaint) [inline build.gradle]: `api 'org.apache.commons:commons-lang3'`

Comment (mcornaton) [reply build.gradle]: Good catch! Please see https://github.com/iExecBlockchainComputing/iexec-common/pull/375/commits/403ab62690d180bacb6666c1191e6aff4d57387a.

Comment (james-toussaint) [reply build.gradle]: Thanks :+1: You can replace double quotes too*

Comment (mcornaton) [reply build.gradle]: Sure. I removed all the double quotes that were useless. Please see https://github.com/iExecBlockchainComputing/iexec-common/pull/375/commits/9cbea402743d87cc205bd4be32aab91c775568da.

Comment (james-toussaint) [reply build.gradle]: Better thanks!

**Review by james-toussaint — APPROVED** [review]
Thanks for the updates!

**Pull request merged**